### PR TITLE
PHP checker

### PR DESCRIPTION
**Pull request by m-1-k-3**
A first checker for php stuff. Only basic syntax checking with php -l is currently done:

image

image

### CODE REVIEW

**Comment by m-1-k-3** [comment]
The check for pylint was removed on a different pull request. Why was this done?

See: https://github.com/e-m-b-a/emba/commit/5d926630a1d35582492b28db7af7252bd0aa529a#r47274266

I have included it again in this pull request.

**Comment by p4cx** [comment]
Please add `php` to our installer script - I checked on my system, it isn't already installed.

**Comment by p4cx** [comment]
> The check for pylint was removed on a different pull request. Why was this done?

I have no idea, but I missed it in the review. Thank you for adding it again.

File: modules/S22_php_check.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#
+# emba comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# emba is licensed under GPLv3
+#
+# Author(s): Michael Messner, Pascal Eckmann
+
+# Description:  Check shell scripts with shellchecker
```
Comment (p4cx) [inline modules/S22_php_check.sh:16]: Just change that line and then I'll merge it. The rest looks good.

**Review by p4cx — CHANGES_REQUESTED** [review]

File: modules/S22_php_check.sh (hunk 1 above)
Comment (m-1-k-3) [reply modules/S22_php_check.sh]: argl ... wasting time

**Review by p4cx — APPROVED** [review]

**Pull request merged**